dh_virtualenv/deployment.py

```diff
@@ -210,20 +210,20 @@ def run_tests(self):
     def find_script_files(self):
         """Find list of files containing python shebangs in the bin directory"""
         command = ['grep', '-l', '-r',
-                   '-e', r'^#!.*bin/\(env \)\?{0}'.format(_PYTHON_INTERPRETERS_REGEX),
+                   '-e', r'^#!\(python\|.*bin/\(env \)\?{0}\)'.format(_PYTHON_INTERPRETERS_REGEX),
```

Comment:
This should also use `{0}`, not just literal `python`.

Reply:
I assume it's the same story for `{names}` in `fix_shebangs` down below, right?